app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -148,43 +165,59 @@ export class LabEntryForm extends PureComponent {
             {this.renderLabDraftOrder()}
           </div>
           <div className="order-form-wrapper">
-            <form className="lab-form simple-form-ui">
-              {this.showFieldSet()}
-            </form>
+            <form className="lab-form simple-form-ui">{this.showFieldSet()}</form>
           </div>
         </div>
         <br />
         {renderActiveOrders()}
```

Comment:
`renderActiveOrders` is still making use of the mock data from `./ordersHistoryMockData`, I am not sure this is supposed to be.

Reply:
This PR has nothing to do with Active Orders, This PR works with Past orders (history)